### PR TITLE
feature/bash

### PR DESCRIPTION
**Pull request by MaryQuiroz**

### CODE REVIEW

File: staff/Mary-Quiroz/bash/README.md
Comment (fercostacastro) [inline staff/Mary-Quiroz/bash/README.md:1]: the explanation could be in English

**Comment by joelsrzalvarez** [comment]
I think you could add more commands 👍

File: staff/mary-quiroz/bash/README.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+## git init (Initializes the new git repository).
+
+```sh
+$ git init
+```
+
+## pwd (path to working directory)
+
+```sh
+It shows us the current folder we are in.
```
Comment (Barrufat) [inline staff/mary-quiroz/bash/README.md]: Each command definition and examples should be written outside the back ticks 🤯

Comment (MaryQuiroz) [reply staff/mary-quiroz/bash/README.md]: done!

File: staff/Mary-Quiroz/bash/README.md
Comment (MaryQuiroz) [reply staff/Mary-Quiroz/bash/README.md:1]: done!

**Comment by manuelbarzi** [comment]
remove commit with arrays

File: staff/mary-quiroz/bash/README.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+```sh
+$ pwd
+```
+
+## mkdir (Allows us to create folders)
+
+```sh
+$ mkdir
+    ej: mkdir newfolder
```
Comment (manuelbarzi) [inline staff/mary-quiroz/bash/README.md]: remove `ej`(emplos) from code blocks

**Pull request merged**